src/core/seat/input-manager.hpp

```diff
@@ -92,8 +93,10 @@ class input_manager
         /* TODO: move this in a wf_keyboard struct,
          * This might not work with multiple keyboards */
         bool in_mod_binding = false;
+        uint32_t mod_binding_key = 0; /* e.g. whether L or R Shift caused a <shift> binding */
```

Comment:
Comment should say "The keycode which triggered the modifier binding". Also I think we no longer need `in_mod_binding`, if `mod_binding_key = 0` then this means we have no modifier binding.